kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiTBinaryFrontendService.scala

```diff
@@ -94,6 +97,10 @@ final class KyuubiTBinaryFrontendService(
 
       respConfiguration.put(KYUUBI_SESSION_ENGINE_LAUNCH_SUPPORT_RESULT, true.toString)
 
+      // KYUUBI-6338: Set default fetch size
```

Comment:
link a Hive ticket instead of Kyuubi here, with brief words to explain why we should do it.